### PR TITLE
Enhancement: Run php-cs-fixer on Travis

### PR DESCRIPTION
**Pull request by localheinz**
This PR

* [x] requires `fabpot/php-cs-fixer`
* [x] adds a simple configuration, using the PSR-2 level
* [x] runs `php-cs-fixer` on Travis, against PHP5.6 only
* [x] adds a composer script to simplify running `php-cs-fixer`
* [x] runs `composer run-script cs`

Somewhat related to #82.

:information_desk_person: For reference, see https://github.com/FriendsOfPHP/PHP-CS-Fixer/tree/1.11.

### CODE REVIEW

File: src/Eris/Generator/ContainmentCheckGenerator.php [hunk 1]
```diff
@@ -1,8 +1,9 @@
 <?php
```
Comment (giorgiosironi) [inline src/Eris/Generator/ContainmentCheckGenerator.php]: this would vanish after rebasing on master

Comment (localheinz) [inline src/Eris/Generator/ContainmentCheckGenerator.php]: Rebasing this second!

**Comment by localheinz** [comment]
@giorgiosironi 

Rebased!

File: cs_fixer_tmp_wCykxr [hunk 2]
```diff
@@ -0,0 +1,30 @@
+<?php
```
Comment (giorgiosironi) [inline cs_fixer_tmp_wCykxr]: accidentally committed file?

File: .php_cs [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<?php
+
+$finder = Symfony\CS\Finder\DefaultFinder::create()->in(__DIR__);
+
+return Symfony\CS\Config\Config::create()
+    ->setUsingCache(true)
+    ->level(Symfony\CS\FixerInterface::PSR2_LEVEL)
```
Comment (giorgiosironi) [inline .php_cs]: note: this PR contains the decision to be PSR-2 compliant, which is useful

File: .travis.yml [hunk 4]
```diff
@@ -6,11 +6,14 @@ env:
...
-  - 5.5
-  - 5.6
-  - 7.0
-  - hhvm
+matrix:
+  fast_finish: true
+  include:
+    - php: 5.5
+    - php: 5.6
+      env: WITH_CS=true
```
Comment (giorgiosironi) [inline .travis.yml]: why particularly on 5.6? I get that it's only necessary to run once, any input on how to choose?

Comment (localheinz) [inline .travis.yml]: @giorgiosironi 

I'm not entirely sure, but I don't think `fabpot/php-cs-fixer` supports fixing PHP7 issues yet, so I figured the best idea is to run it on the most recent version smaller than `7`.

**Comment by giorgiosironi** [comment]
Two things:

- remove the accidentally committed file at the root of the project
- then this PR goes at the end of the queue with PSR-4, since it's going to create so many conflicts with other ones. Need to merge all the previously opened ones first

**Comment by localheinz** [comment]
@giorgiosironi 

Done!

**Pull request merged**